crates/matrix-sdk/src/room/mod.rs

```diff
@@ -403,6 +403,18 @@ impl Room {
         IdentityStatusChanges::create_stream(self.clone()).await
     }
 
+    /// Mark this user as "pinned" meaning that we have seen their cryptographic
+    /// identity before.
+    #[cfg(feature = "e2e-encryption")]
+    pub async fn pin_user_identity(&self, user_id: &UserId) -> Result<()> {
+        let x = self.client.encryption().get_user_identity(user_id).await?;
+
+        match x {
```

Comment:
nit: either inline `x`, or give it a better name?

Reply:
Thanks, sorry I realised this the same time as you did and force-pushed an update.